rllib/examples/rock_paper_scissors_multiagent.py

```diff
@@ -202,11 +183,14 @@ def entropy_policy_gradient_loss(policy, model, dist_class, train_batch):
         loss_fn=entropy_policy_gradient_loss)
     EntropyLossPG = PGTrainer.with_updates(
         name="EntropyPG", get_policy_class=lambda _: EntropyPolicy)
-    run_heuristic_vs_learned(use_lstm=True, trainer=EntropyLossPG)
+    run_heuristic_vs_learned(args, use_lstm=True, trainer=EntropyLossPG)
 
 
 if __name__ == "__main__":
-    # run_same_policy()
-    # run_heuristic_vs_learned(use_lstm=False)
-    run_heuristic_vs_learned(use_lstm=False)
-    # run_with_custom_entropy_loss()
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--stop", type=int, default=1000)
+    args = parser.parse_args()
+    run_same_policy(args)
+    run_heuristic_vs_learned(args, use_lstm=True)
+    run_heuristic_vs_learned(args, use_lstm=False)
+    run_with_custom_entropy_loss(args)
```

Comment:
Hmm I can see this being a bit confusing to run as an example since there are four different runs in the output.

Could we at least add print()s in between?

Reply:
Yeah, I just fixed that test case. For some reason, 3 of these have always been commented out and weren't working. I'll add the print()s.

Reply:
Done.